### DISCUSSION TITLE
Move files into VueJS project folder

### DISCUSSION THREAD

**Discussion by ActuallySeb** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#45] [Ideas]

**Comment by david-baez-bravo** [comment]
This issue orginally said:

After creating the VueJS project folder, files need to be placed there in order to use the framework

Objective: Move folders into the vue-project folder

Difficulty: 2
Urgency: 4

**Comment by david-baez-bravo** [comment]
This issue was to simple and could be combine d with another Issue and that is what was done
